lib/src/main/java/uk/gov/di/ipv/cri/passport/library/config/ConfigurationVariable.java

```diff
@@ -1,29 +1,26 @@
 package uk.gov.di.ipv.cri.passport.library.config;
 
 public enum ConfigurationVariable {
-    AUTH_CODE_EXPIRY_CODE_SECONDS("/%s/credentialIssuers/ukPassport/self/authCodeExpirySeconds"),
-    BACKEND_SESSION_TTL("/%s/credentialIssuers/ukPassport/self/backendSessionTtl"),
-    DCS_POST_URL_PARAM("/%s/credentialIssuers/ukPassport/dcs/postUrl"),
-    DCS_ENCRYPTION_CERT("/%s/credentialIssuers/ukPassport/dcs/encryptionCertForPassportToEncrypt"),
-    DCS_SIGNING_CERT("/%s/credentialIssuers/ukPassport/dcs/signingCertForPassportToVerify"),
-    DCS_TLS_INTERMEDIATE_CERT("/%s/credentialIssuers/ukPassport/dcs/tlsIntermediateCertificate"),
-    DCS_TLS_ROOT_CERT("/%s/credentialIssuers/ukPassport/dcs/tlsRootCertificate"),
-    JAR_ENCRYPTION_KEY_ID("/%s/credentialIssuers/ukPassport/self/jarKmsEncryptionKeyId"),
-    JAR_KMS_PUBLIC_KEY("/%s/credentialIssuers/ukPassport/self/jarKmsEncryptionPublicKey"),
-    MAX_JWT_TTL("/%s/credentialIssuers/ukPassport/self/maxJwtTtl"),
-    MAXIMUM_ATTEMPT_COUNT("/%s/credentialIssuers/ukPassport/self/maximumAttemptCount"),
-    PASSPORT_CRI_CLIENT_AUDIENCE("/%s/credentialIssuers/ukPassport/self/audienceForClients"),
-    PASSPORT_CRI_CLIENT_AUTH_MAX_TTL("/%s/credentialIssuers/ukPassport/self/maxJwtTtl"),
+    AUTH_CODE_EXPIRY_CODE_SECONDS("/%s/AuthCodeExpirySeconds"),
+    SESSION_TTL("/%s/SessionTtl"),
+    DCS_POST_URL_PARAM("/%s/credentialIssuers/ukpassport/DCS/PostUrl"),
```

Comment:
I'm guessing some of these changes means your going to update this in our config repo as well?

I think there are a few ssm params here that aren't being created in the new template file, so the new param names will need adding in our config repo.

Reply:
@smsgds this is one we will need to pickup theres a thread about it in the CRI LIME chat